src/app/globals.css

```
@@ -1,33 +1,104 @@
 :root {
-  --color-gray-98: hsl(0deg 0% 98%);
-  --color-gray-90: hsl(0deg 0% 90%);
-  --color-gray-80: hsl(0deg 0% 80%);
-  --color-gray-70: hsl(0deg 0% 70%);
-  --color-gray-40: hsl(0deg 0% 40%);
-  --color-gray-30: hsl(0deg 0% 30%);
-  --color-gray-20: hsl(0deg 0% 20%);
-  --color-gray-16: hsl(0deg 0% 16%);
-  --color-gray-12: hsl(0deg 0% 12%);
-  --color-gray-10: hsl(0deg 0% 10%);
-
-  --color-default-background: var(--color-gray-10);
-  --color-default-foreground: var(--color-gray-98);
-
-  --color-primary: hsl(41, 92%, 61%);
-  --color-primary-fade: hsl(120deg 50% 96%);
-  --color-primary-lighter: hsl(41, 92%, 43%);
-  --color-primary-darker: hsl(41, 92%, 55%);
-  --color-primary-opposite: var(--color-gray-10);
-  --color-secendry: hsl(256, 98%, 37%);
-
-  --border-radius: 0.875rem;
-
-  --full-width: 75rem;
-  --full-width-padding-inline: calc(max(100% - var(--full-width), 2rem) / 2);
-
-  --animation-duration-fast: 0.1s;
-  --animation-duration-normal: 0.5s;
-  --animation-duration-slow: 1s;
+  .dark {
+    --color-gray-98: hsl(0deg 0% 98%);
+    --color-gray-90: hsl(0deg 0% 90%);
+    --color-gray-80: hsl(0deg 0% 80%);
+    --color-gray-70: hsl(0deg 0% 70%);
+    --color-gray-40: hsl(0deg 0% 40%);
+    --color-gray-30: hsl(0deg 0% 30%);
+    --color-gray-20: hsl(0deg 0% 20%);
+    --color-gray-16: hsl(0deg 0% 16%);
+    --color-gray-12: hsl(0deg 0% 12%);
+    --color-gray-10: hsl(0deg 0% 10%);
+
+    --color-default-background: var(--color-gray-10);
+    --color-default-foreground: var(--color-gray-98);
+
+    --color-primary: hsl(41, 92%, 61%);
+    --color-primary-fade: hsl(120deg 50% 96%);
```

همه جا از یه Syntax یکسان استفاده کنید

```suggestion
    --color-primary: hsl(41, 92%, 61%);
    --color-primary-fade: hsl(120, 50%, 96%);
```